wake-word-voice-assistant/m5stack-atom-echo.yaml

```diff
@@ -1 +1,352 @@
-<<: !include ../voice-assistant/m5stack-atom-echo.yaml
+substitutions:
+  name: m5stack-atom-echo
+  friendly_name: M5Stack Atom Echo
+  micro_wake_word_model: okay_nabu # alexa, hey_jarvis and hey_mycroft are also supported
```

Comment:
Shouldn't this be an array, because it is possible to use multiple wake words!?!

Reply:
this is a good point. However it seems substitutions don't support lists. I tried and failed in f0a8943

Reply:
What is more, the M5 stack only supports two models (including VAD and VAD is enabled). To support a second it has to be overclocked, and I'm not sure that is a good idea in default firmware.

Reply:
I'm aware that different wake words starting different conversation agents is a planned feature - but it is not yet implemented, as such there is no great use two wake words.
When it does become a feature we can open a new PR discussing either removing the VAD model or introducing overclocking.